File: Assets/Scripts/Model/PlayerModel.java
package Model;

public enum PlayerModel {
    X(0,0),
    O(0,0);

    int Moves;
    int Wins;

    PlayerModel(int Moves, int Wins){
        this.Moves = Moves;
        this.Wins = Wins;
    }

    public int GetMoves(PlayerModel thisPlayerModel){
        return thisPlayerModel.Moves;
    }
    public int GetWins(PlayerModel thisPlayerModel){
        return thisPlayerModel.Wins;
    }

    public void AddMoves(PlayerModel thisPlayerModel){
        thisPlayerModel.Moves++;
    }
    public void AddWins(PlayerModel thisPlayerModel){
        thisPlayerModel.Wins++;
    }

}

File: Assets/Scripts/Model/PlayerModel.cs
using System.Collections.Generic;
using Unity.Mathematics;
using UnityEngine;

namespace Model
{
    public class PlayerModel : MonoBehaviour
    {
        public char[,] moves = {{' ', ' ', ' '}, {' ', ' ', ' '}, {' ', ' ', ' '}};
        public int TotalMovesPlayed = 0;

    private int Wins = 0;
        
        public void AddMoves(int i, int j, int playerID){
            TotalMovesPlayed++;
            moves[i,j] = playerID == 0 ? 'X' : 'O';
        }
        public void AddWins(){
            Wins++;
        }
        
        public char[,] GetMoves(){
            return moves;
        }
        public int GetWins(){
            return Wins;
        }
        
        public int GetTotalMoves(){
            return TotalMovesPlayed;
        }
    }
}